pygen/pygen_src/riscv_asm_program_gen.py

```diff
@@ -437,15 +437,35 @@ def init_floating_point_gpr_with_dpf(self, int_floating_gpr):
         self.instr_stream.extend((li_instr0, slli_instr, li_instr1, or_instr, fmv_instr))
 
     # Get a random single precision floating value
+    @staticmethod
+    def get_randselect0(addr_range1, addr_range2):
```

Comment:
why is it get_randselect"0"? Can we use a better name?

Reply:
I see there is another version of this function for 64 bits. Can you please combine the two methods?